src/index.html

```diff
@@ -2,10 +2,24 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+    <title>To do List</title>
+    <script src="https://kit.fontawesome.com/f51be44a58.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;1,100&display=swap" rel="stylesheet">
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <div class="all-container">
+    <div class="container">
+        <div class="card-layout">
+            <div class="card-title"> <p>Today`s To Do</p> <i class="fa-solid fa-rotate"></i> </div>
+            <div class="sub-title"><em> Add to your list... </em></div>
```

Comment:
- [ ] You did well 😃 at this milestone, but I see here that you make the text add todo static and we want it to be input to let the user can write the todo, so kindly 😊 try to add the input here to be like the original design to let the users can write their todos

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| ![Recording 2023-02-22 at 13 53 26](https://user-images.githubusercontent.com/83360636/220612767-68af087c-9ed2-4f02-9697-26639f087159.gif)   |     ![Recording 2023-02-22 at 13 54 37](https://user-images.githubusercontent.com/83360636/220612973-4a8ccb22-0ed7-4903-8c85-390e162e872d.gif)   |

Reply:
Dear @osamaashraf6 thank you for your constructive comments. but I am little bit confusing with the tamplate, this the first setup the next day add and remove project will be done next to this project. may I get some inforamtion please?